code/__DEFINES/research.dm

```diff
@@ -30,16 +30,15 @@
 	TECHWEB_POINT_TYPE_GENERIC = "General Research",\
 	)
 
-/// The maximum amount of cash you can get from toxins experiments.
+/// The maximum amount of cash you can get from ordnance experiments.
 #define TECHWEB_BOMB_CASHCAP 50000
-/// The range required to generate any cash from toxins experiments.
+/// The range required to generate any cash from ordnance experiments.
 #define TECHWEB_BOMB_MIN_RANGE (GLOB.MAX_EX_LIGHT_RANGE)
-/// The range required to produce the maximum amount of cash from toxins experiments.
+/// The range required to produce the maximum amount of cash from ordnance experiments.
 #define TECHWEB_BOMB_MAX_RANGE 300
-/// A constant used to scale the cash produced by toxins experiments.
+/// A constant used to scale the cash produced by ordnance experiments.
 #define TECHWEB_BOMB_SCALE_CONST 300000
-/// A constant used to scale the cash produced by toxins experiments.
-#define TECHWEB_BOMB_SCALE_DIVISOR 1500
+/// A constant used to scale the cash produced by ordnance experiments.t#define TECHWEB_BOMB_SCALE_DIVISOR 1500
```

Comment:
You accidentally moved the define into the comment

Reply:
Yep yep yep, fixing it